src/app/store/store.module.ts

```diff
@@ -8,7 +8,7 @@ import { SharedModule } from 'app/shared/shared.module';
 import { StoreComponent } from './store.component';
 import { ExamplesModule } from '../examples/examples.module';
 import { ProductDetailComponent } from './product-detail/product-detail.component';
-import { StoreService } from './store/store.service';
+// import { StoreService } from './store/store.service';
```

Comment:
Since you moved this to the shared module, we'll want to remove these commented out lines and import sharedmodule here. (if it isn't already imported here)